resources/js/features/game-list/components/SimilarGameSuggestionsMainRoot/SimilarGameSuggestionsMainRoot.tsx

```diff
@@ -0,0 +1,50 @@
+import { router } from '@inertiajs/react';
+import { type FC, memo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuDices, LuUser } from 'react-icons/lu';
+
+import { BaseButton, baseButtonVariants } from '@/common/components/+vendor/BaseButton';
+import { GameHeading } from '@/common/components/GameHeading';
+import { InertiaLink } from '@/common/components/InertiaLink';
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+import { GameSuggestionsDataTable } from '../GameSuggestionsDataTable';
+
+export const SimilarGameSuggestionsMainRoot: FC = memo(() => {
+  const { sourceGame } = usePageProps<App.Platform.Data.GameSuggestPageProps>();
+
+  const { t } = useTranslation();
+
+  const game = sourceGame as App.Platform.Data.Game;
+
+  const handleReload = () => {
+    router.reload({ only: ['paginatedGameListEntries'] });
+  };
+
+  return (
+    <div>
+      <GameHeading game={game}>{t('Game Suggestions')}</GameHeading>
+
+      <div className="flex flex-col gap-2">
+        <div className="flex justify-end rounded bg-embed p-2">
+          <div className="flex gap-2">
+            <InertiaLink
+              href={route('game.suggestions.personalized')}
+              className={baseButtonVariants({ size: 'sm', className: 'gap-1' })}
+            >
+              <LuUser className="size-4" />
+              {t('See games picked just for you')}
+            </InertiaLink>
+
+            <BaseButton onClick={handleReload} size="sm" className="group gap-1">
+              <LuDices className="size-4 transition-transform duration-100 group-hover:rotate-12" />
+              <span className="hidden sm:inline">{t('Roll again')}</span>
+            </BaseButton>
+          </div>
+        </div>
+
+        <GameSuggestionsDataTable />
```

Comment:
Not sure how easy it would be to remove the relatedGame icon from every row, as it's implied by the page header.
![image](https://github.com/user-attachments/assets/4df19146-ef22-4a15-9f0c-c7ead1a55208)

Reply:
Took some creativity, but should be updated in latest:
![Screenshot 2025-01-16 at 8 18 31 PM](https://github.com/user-attachments/assets/6a73845c-9927-466e-97e5-22128b3ae13b)

The only wording I'm not so confident in is the similar beaten/mastered verbiage, ie: "Also mastered by same players". Maybe it actually sounds okay and it only sounds weird to me because I've been reciting it in my head over and over.

Reply:
I think it's okay. Maybe it might be less weird if you got rid of the Also?

Reply:
Yeah, with a fresh set of eyes, removing "also" from those two phrases sounds better. Done in latest.